deploy/02-webhook.permissions.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: webhook-validation-cr
+rules:
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+    - validatingwebhookconfigurations
+  verbs:
+    - list
+    - patch
```

Comment:
Is there any use case that requires this `patch` permission on VWC

Reply:
The initContainer needs `patch` to inject the caBundle.

Reply:
What we could do is limit the resourceNames to some list of specified (or computed, from yaml) VWC. I can spend some effort on that if you think it's worthwhile, otherwise, all this service account will be able to edit all VWC.

Reply:
I think it's ok for now, just confirming since I couldn't think of a use case for patch